src/main/java/com/thealgorithms/bitmanipulation/HighestSetBit.java

```diff
@@ -12,7 +12,7 @@ public final class HighestSetBit {
     private HighestSetBit() {
     }
 
-    public final static Optional<Integer> findHighestSetBit(int num) {
+    public static final Optional<Integer> findHighestSetBit(int num) {
```

Comment:
```suggestion
    public static Optional<Integer> findHighestSetBit(int num) {
```

This `final` can be removed.